backends/graphics/sdl/sdl-graphics.cpp

```diff
@@ -513,10 +513,6 @@ void SdlGraphicsManager::initImGui(void *glContext) {
 	assert(!_imGuiReady);
 	_imGuiInited = false;
 
-	if (!glContext) {
-		return;
-	}
-
```

Comment:
Why has this check been removed?

Reply:
This essentially changes the SdlGraphicsMgr class to accept a nullptr glContext. Later, ImGui_ImplSDL2_InitForOpenGL is then called with a null context which is exactly what we need for the non-GL backend to be picked under the hood (this was discussed on Discord w/ the rest of the team).